cpp/src/cider/exec/nextgen/jitlib/llvmjit/LLVMJITValue.cpp

```diff
@@ -39,63 +39,69 @@ JITValue& LLVMJITValue::assign(JITValue& value) {
 }
 
 JITValuePointer LLVMJITValue::andOp(JITValue& rh) {
+  CHECK(rh.getValueTypeTag() == getValueTypeTag());
```

Comment:
if CHECK failed, this macro will abort the process, Is that expected?

Reply:
Yes, exposing errors explicitly.